Skills.md

```diff
@@ -1,61 +1,80 @@
+## Challenge Rules
+Challenges are handled by rolling three dice, associated with three elements. There is one major element, and two minor.
+A success requires rolling greater than a threshold on the major element, and a least one of the minor elements. A failure
+on only one of the minor elements results in a minor negative side effect, the details of which are left up to the GM,
+but where possible associated with the element that failed.
+
+A challenge occurs whenever the GM asks a player to roll for a challenge. The GM will say something like 'roll a fire, water,
+earth test, threshold 8'. This means that the player must roll a die for each of the elements, trying to get 6 or more for
+each element. The die may be different for each element, and is determined by the total number of gems of the given element
+that the player owns. The player then rolls the 3 dice, and adds a value to each die, depending on the number of gems
+in the sphere matching the given element. Add 1 for every non-matching, and 2 for every matching gem. This happens for
+each element independantly. A success is achieved if, and only if, the major element passes, and at least one of the
+minor elements passes.
 
 Dice for gems
 
-| Number of Gems | Die |
-| -------------- | --- |
-| 0*             | d10 |
-| 1              | d8  |
-| 2              | d6  |
-| 3+**           | d4  |
+| Number of Gems Owned | Die |
+| -------------------- | --- |
+| 0*                   | d6  |
+| 1                    | d8  |
+| 2                    | d10 |
+| 3+**                 | d12 |
 * Note that with the current rule set, 0 gems of a colour is not possible. However, there may be reason to add it as a result
 of an in game effect, so it is added here
 ** The rules currently do not allow more than 3 gems. If this becomes a possiblity for some reason, and a player has more than
 3 gems of a colour, they still only roll a d4.
 
-Threshold Modifiers
-
-| Level | Modifier |
-| ----- | -------- |
-| Trivial   | +3   |
-| Very Easy | +2   |
-| Easy      | +1   |
-| Average   |  0   |
-| Hard      | -1   |
-| Very Hard | -2   |
-| Nigh Impossible | -3 |
-
-Note that the threshold can never be below 1. If characters shouldn't be able to do something, just say so, and don't bother
-with a test. It is reasonable to say, for example, "You have no experience with this; you try for a while, but fail to accomplish
-anything".
-
 For example:
 Robert wants to ask a villager for help. The GM asks how he wants to do so. Robert says that he wants to plea, pulling on the
 villagers heart strings. The GM doesn't see any physical impact from the action, and determines the Robert is trying to actively
-change the villagers mind, so he calls for an easy 'lww' (Lightning, Water, Water) test, as lightning represents change,
+change the villagers mind, so he calls for a 'Water, Lightning, Water, threshold 8' test, as lightning represents change,
```

Comment:
I think a description of what each element means and how they relate to the skill checks being made would be an important addition, both to provide GMs some basis for making calls on elements, and for players to assist them in their assumptions. Providing examples for a few different typical skill checks is probably a good idea too (perception, athletics, persuasion and stealth are good candidates). That probably belongs to a separate PR.

Reply:
Agreed. I'll be working on that soon.